miq_server_leak_discovery/simple_memory_monitor.rb

```diff
@@ -1,6 +1,31 @@
 require 'sys-proctable'
 
+# coma separated values - easier to read
```

Comment:
For what it is worth, the addition of a digit was enough for my purposes to know if there was heavy growth.  Didn't need to know the order of magnitude in most cases.